scripts/hudson/narayana.sh

```diff
@@ -1050,7 +1050,7 @@ MainClassPatterns="org.jboss.jbossts.qa com.arjuna.ats.arjuna.recovery.RecoveryM
 kill_qa_suite_processes $MainClassPatterns
 
 export MEM_SIZE=1024m
-[ -z ${MAVEN_OPTS+x} ] && export MAVEN_OPTS="-Xms$MEM_SIZE -Xmx$MEM_SIZE"
+[ -z ${MAVEN_OPTS+x} ] && [ "$_jdk" -lt 17 ] && export MAVEN_OPTS="-Xms$MEM_SIZE -Xmx$MEM_SIZE"
```

Comment:
@mayankkunwar does Xms and Xmx not work on JDK 17 either? Or is the protection here for some other reason? I recall is chatting about checking the $_jdk but IIRC wasn't it when something like MaxPermSize was provided? Sorry if I misunderstood something about this.